shared/estate.js

```diff
@@ -0,0 +1,23 @@
+export function isEstate(asset) {
+  return !!asset.parcels
+}
+
+export function getEstateByParcel(parcel, estates) {
+  return Object.keys(estates)
+    .map(estateId => estates[estateId])
+    .find(estate => estate.id === parcel.owner)
+}
+
+export function areOnSameEstate(parcels) {
```

Comment:
Maybe, I don't like very much the name either but I wanted the function to explicity say the is validating that the parcels are all on the same estate, `belongToEstate` might be a little bit confusing, because I imagine that it validates that the parcels are on estates, but not at the same one, maybe `belongToSameEstate`?

Reply:
I like `belongToSameState`!